packages/services/hmi-server/src/main/resources/application.properties

```diff
@@ -50,7 +50,8 @@ mit/mp-rest/url=https://mit-proxy.staging.terarium.ai
 github/mp-rest/url=https://api.github.com
 jsdelivr/mp-rest/url=https://cdn.jsdelivr.net
 mira-dkg/mp-rest/url=http://34.230.33.149:8771
-pdf/mp-rest/url=http://localhost:5000
+%dev.pdf/mp-rest/url=http://localhost:5000
+%prod.pdf/mp-rest/url=https://hmi-extraction.staging.terarium.ai:5000/
```

Comment:
I wonder if we should do like simulation-service and only use staging, unless needed to run locally.

Reply:
Yeah I was wondering the same thing.  I'm good either/or.  lmk.

Reply:
can I make this switch later?  caught in this chicken/egg paradox right now since we don't have anything on staging yet.